### DISCUSSION TITLE
Documentation ?

### DISCUSSION THREAD

**Discussion by ValentinH** [johnlindquist/kit#660] [Q&A]
I might have missed it but is there already a documentation? For example, I'd like to be able to look at available commands 🙂

**Comment by johnlindquist** [comment]
The app has a "Docs" tabs which loads in the "Docs" category from these GitHub discussions.

When I update the Github discussions Docs, a new json file is generated that the app checks for daily. This allows the docs to stay up-to-date based on user feedback and questions.

**Reply by blackhaj** [reply]
Hey @johnlindquist - I am just getting started but loving Kit.

One small piece of feedback on this theme - I have been finding the current docs structure a little hard to navigate (for example, I wasn't sure where to find the function to get clipboard text and it took a lot of searching for best guesses in the Github docs page to find it).

Do you have any plans to create a more centralized API docs page where users can find the full list of helpers and their function signatures etc? Or is there a better way to find this info that I am missing?

I plan to create my own 'docs' to help me as I learn the API which I would be very happy to share this if it was useful. If you have guidance on the best way to do this, I would love to hear it.

Huge thanks for the amazing tool

**Reply by johnlindquist** [reply]
@blackhaj Yeah, docs will be a big focus going forward. I'll get everything tagged/named/orgnized/etc better to make them more searchable. Even considering making it possible to search the "body"/examples of the docs page from the app, but that would require a bit of re-working. Also definitely want to add examples for each API.

Would love to see any notes you put together. I definitely want to use GitHub discussions as the "CMS" for the docs so that people (with a trusted GitHub identity) can leave feedback, but I'm open to ideas.

**Reply by blackhaj** [reply]
Thanks @johnlindquist - great to hear docs will be a focus. Searching the body of the examples would be very helpful too.

I will be sure to share what I come up with in case it is helpful. Thanks again for a great app!
